bazel_tools/ghc-lib/version.bzl

```diff
@@ -9,7 +9,7 @@ GHC_LIB_PATCHES = [
 ]
 
 GHC_REPO_URL = "https://github.com/digital-asset/ghc"
-GHC_REV = "c119f3bdad49336c2b8d17e04d7070ef419132b8"
+GHC_REV = "bbe22fa527a6db95534c1fe70163bcfff8a88eb6"
```

Comment:
please remember to update after merging https://github.com/digital-asset/ghc/pull/135